include/html2bbcode.php

```diff
@@ -313,15 +313,69 @@ function ($matches) use (&$codeblocks) {
 	// Restore code blocks
 	$message = preg_replace_callback('#\[codeblock-([0-9]+)\]#iU',
 		function ($matches) use ($codeblocks) {
-            $return = '';
-            if (isset($codeblocks[intval($matches[1])])) {
-                $return = $codeblocks[$matches[1]];
-            }
+	    $return = '';
+	    if (isset($codeblocks[intval($matches[1])])) {
+		$return = $codeblocks[$matches[1]];
+	    }
 			return $return;
 		}
 	, $message);
 
 	$message = trim($message);
 
+	if ($basepath != '') {
+		$message = AddHostname($message, $basepath);
+	}
+
 	return $message;
 }
+
+/**
+ * @brief Sub function to complete incomplete URL
+ *
+ * @param array $matches Result of preg_replace_callback
+ * @param string $basepath Basepath that is used to complete the URL
+ *
+ * @return string The expanded URL
+ */
+function AddHostnameSub($matches, $basepath) {
```

Comment:
Can you please use `camelCase` for function names?

Reply:
Isn't this?

Reply:
`camelCase` doesn't have an uppercase first letter. `StudlyCaps` or `PascalCase` do.

Reply:
This is confusing :-)